tools/create_triage_tickets.py

```diff
@@ -162,6 +160,11 @@ def main(arg):
              '{signature_type: {root_issue: message}}',
         default='./triage_resolving_filters.json',
     )
+    parser.add_argument(
```

Comment:
The argument name is misleading. because we are assigning only closed tickets
Just add it as a constant
.